### PR TITLE
Refactor constants to separate file

### PR DESCRIPTION
**Pull request by fartariatomas**

### CODE REVIEW

**Review by pedrobento988 — APPROVED** [review]

**Review by carolinaloureiro — APPROVED** [review]
too many blank spaces but ok

File: kafka-connect-hbase/src/main/scala/com/datamountaineer/streamreactor/connect/hbase/config/HbaseSinkConfigConstants.scala [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  val EXPORT_ROUTE_QUERY = "connect.hbase.sink.kcql"
+  val EXPORT_ROUTE_QUERY_DOC = "KCQL expression describing field selection and routes."
+
+  val ERROR_POLICY = "connect.hbase.sink.error.policy"
+  val ERROR_POLICY_DOC: String =
+    """
+      |Specifies the action to be taken if an error occurs while inserting the data.
+      |There are two available options:
+      |NOOP - the error is swallowed
+      |THROW - the error is allowed to propagate. \n
```
Comment (pedrobento988) [inline kafka-connect-hbase/src/main/scala/com/datamountaineer/streamreactor/connect/hbase/config/HbaseSinkConfigConstants.scala]: please remove \n

File: kafka-connect-hbase/src/main/scala/com/datamountaineer/streamreactor/connect/hbase/config/HbaseSinkConfigConstants.scala [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+object HbaseSinkConfigConstants {
+  val COLUMN_FAMILY = "connect.hbase.sink.column.family"
+  val COLUMN_FAMILY_DOC = "The hbase column family."
+
+  val EXPORT_ROUTE_QUERY = "connect.hbase.sink.kcql"
+  val EXPORT_ROUTE_QUERY_DOC = "KCQL expression describing field selection and routes."
+
+  val ERROR_POLICY = "connect.hbase.sink.error.policy"
+  val ERROR_POLICY_DOC: String =
+    """
```
Comment (pedrobento988) [inline kafka-connect-hbase/src/main/scala/com/datamountaineer/streamreactor/connect/hbase/config/HbaseSinkConfigConstants.scala]: Your are inserting a new empty line

File: kafka-connect-hbase/src/main/scala/com/datamountaineer/streamreactor/connect/hbase/config/HbaseSinkConfigConstants.scala [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      |Specifies the action to be taken if an error occurs while inserting the data.
+      |There are two available options:
+      |NOOP - the error is swallowed
+      |THROW - the error is allowed to propagate. \n
+      |RETRY - The exception causes the Connect framework to retry the message. The number of retries is based on
+      |The error will be logged automatically"
+    """.stripMargin
```
Comment (pedrobento988) [inline kafka-connect-hbase/src/main/scala/com/datamountaineer/streamreactor/connect/hbase/config/HbaseSinkConfigConstants.scala]: Your are inserting a new empty line

**Review by pedrobento988 — CHANGES_REQUESTED** [review]

**Review by pedrobento988 — APPROVED** [review]

**Pull request merged**